include/clang/Parse/Parser.h

```diff
@@ -449,7 +458,7 @@ class Parser : public CodeCompletionHandler {
     if (Tok.getKind() == tok::l_brace)
       ++BraceCount;
     else if (BraceCount)
-      --BraceCount;     // Don't let unbalanced }'s drive the count negative.
+      --BraceCount;     // Don't let unbalanced '}'s drive the count negative.
```

Comment:
Could you remove this change?   It is unrelated to implementing unchecked scopes?